Classes/Backend/SolrModule/IndexQueueModuleController.php

```diff
@@ -114,6 +114,32 @@ public function initializeIndexQueueAction() {
 	}
 
 	/**
+	 * Removes all errors in the index queue list. So that the items can be indexed again.
+	 *
+	 * @return void
+	 */
+	public function flushLogErrorsAction() {
+		/** @var DatabaseConnection $database */
+		$database = $GLOBALS['TYPO3_DB'];
+		$flushResult = $database->exec_UPDATEquery('tx_solr_indexqueue_item', 'errors NOT LIKE ""', ['errors' => '']);
+
+		$message = 'All errors has been flushed';
```

Comment:
To be honest i did not used locallang labels because the whole module is not using it ;). But it would be nice for sure.